### PR TITLE
PrSM output for top-down

### PR DESCRIPTION
**Pull request by zrolfs**
-Changed PSM output to PrSM
-Added unit test for whenever we get around to #2061

### CODE REVIEW

File: TaskLayer/SearchTask/PostSearchAnalysisTask.cs
Comment (trishorts) [inline TaskLayer/SearchTask/PostSearchAnalysisTask.cs:520]: analyteString

File: Test/TopDownTest.cs
Comment (trishorts) [inline Test/TopDownTest.cs:58]: Is this a "TODO"? We have used that terminology in the past. Maybe switch to that. And add an issue.

**Review by trishorts — APPROVED** [review]
This looks good to me. To very minor things to attend to ad libitum.

**Review by rmillikin — APPROVED** [review]

**Review by rmmiller22 — APPROVED** [review]

**Comment by rmmiller22** [comment]
Update the branch and then I will merge in the PR.

**Comment by rmmiller22** [comment]
This pull request is being reverted. Jenkins produced 0 PSMs once it (and another PR) were merged. You can re-open/ re-create this PR- and can be merged again once issues are identified and resolved.

**Pull request merged**
